experimental/PropertyDDS/packages/property-changeset/.eslintrc.js

```diff
@@ -47,6 +47,7 @@ module.exports = {
         "prefer-const": "off",
         "prefer-template": "off",
         "quote-props": "off",
+        "tsdoc/syntax": "off",
```

Comment:
These packages have tons of violations, likely owing to the code's origins as JS. I don't want to muddy this PR with changes for these experimental packages.